src/Compilers/CSharp/Portable/xlf/CSharpResources.de.xlf

```diff
@@ -8620,7 +8540,7 @@ Um die Warnung zu beheben, können Sie stattdessen /reference verwenden (Einbett
       </trans-unit>
       <trans-unit id="ERR_AutoPropertyInitializerInInterface">
         <source>Instance auto-implemented properties inside interfaces cannot have initializers.</source>
-        <target state="needs-review-translation">Automatisch implementierte Eigenschaften in Schnittstellen können keine Initialisierer aufweisen.</target>
+        <target state="translated">Von der Instanz automatisch implementierte Eigenschaften in Schnittstellen können keine Initialisierer aufweisen.</target>
```

Comment:
"Automatisch implementierte Instanz-Eigenschaften"?

Reply:
@jhinder we implemented your suggestions.